composer/helpers/v2/src/UpdateChecker.php

```diff
@@ -75,7 +75,8 @@ public static function getLatestResolvableVersion(array $args): ?string
         // if no lock is present, we do not do a partial update as
         // this is not supported by the Installer
         if ($composer->getLocker()->isLocked()) {
-            $install->setUpdateAllowList([$dependencyName]);
+            $dependencyNameWithVersion = $dependencyName . ':' . $latestAllowableVersion;
```

Comment:
what happens if composer itself fails with this command? Do we get a clear error message surfaced to the user?